.github/workflows/publish.yml

```diff
@@ -1,28 +1,43 @@
-name: Publish packages to npm
+name: Publish
 
 on:
-  push:
-    tags:
-      - 'changesets-*' #temporary, will eventually be '@apollo/composition@2*'
-
+  release:
+    types: [published]
+    
 jobs:
+  slack:
+    name: Publish slack webhook notifying about approval
+    if: "contains(github.event.head_commit.message, 'release:')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send to Slack
+        env: 
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        run: |
+          echo webhook url $SLACK_WEBHOOK
+#         TODO: slack integration
```

Comment:
Is this still a TODO?

Reply:
I'll leave it for now. Either we keep the approval step and have to add slack integration or we get rid of it and have to make other changes.